tools/data/parse_file_list.py

```diff
@@ -378,49 +382,41 @@ def generate_class_index_file():
         with open(class_index_file, 'w') as f:
             content = []
             for class_id, class_name in enumerate(class_list):
+                # like `ClassInd.txt` in UCF-101, the class_id begins with 1
```

Comment:
if class id begins with 1, then num class in the config file should be 52?

same question for ucf101

Reply:
okay never mind, the id subtracts 1 later